### DISCUSSION TITLE
Cannot import a solid/svelte component directly in a page

### DISCUSSION THREAD

**Discussion by 3bdallahz** [ElMassimo/iles#50] [Q&A] [answered]
Hi,
so I'm trying to import a solid component(also tried svelte) in a page instead of the auto imported components and I get and error, interestingly using a component that is auto imported that exports the component I want works
here is an example of what's happening, see file pages/index.vue
https://github.com/abdallahz3/reproduce-issue

**Comment by ElMassimo** [comment, marked as answer]
Hi @abdallahz3!

Thanks for providing a reproduction! This is a [regression](https://github.com/ElMassimo/iles/commit/5dc407857b7ee6e0fe8c838550bca6e79d1fdb52#diff-91961801cff9a0fde9b93307b96f0c5a301351fc1ab0b06311cbac5e848f9e66L342), import paths in islands were no longer being resolved.

As a result, when importing components using an alias (such as `~`) it was not resolved which caused it to work incorrectly during development.

Fixed in [this commit](https://github.com/ElMassimo/iles/commit/f66d14d3e6be805475f5a5a971e859403316e340), will let you know once it's released in a patch version.

**Reply by 3bdallahz** [reply]
Thanks @ElMassimo

**Reply by ElMassimo** [reply]
@abdallahz3 Fixed in [`iles@0.6.5`](https://github.com/ElMassimo/iles/blob/main/packages/iles/CHANGELOG.md#065-2021-11-26), thanks again for reporting! 😃
